identity/handler.go

```diff
@@ -510,26 +543,26 @@ func (h *Handler) update(w http.ResponseWriter, r *http.Request, ps httprouter.P
 	h.r.Writer().Write(w, r, WithCredentialsMetadataAndAdminMetadataInJSON(*identity))
 }
 
-// swagger:parameters adminDeleteIdentity
+// Delete Identity Parameters
+//
+// swagger:parameters deleteIdentity
 // nolint:deadcode,unused
-type adminDeleteIdentity struct {
+type deleteIdentity struct {
 	// ID is the identity's ID.
 	//
 	// required: true
 	// in: path
 	ID string `json:"id"`
 }
 
-// swagger:route DELETE /admin/identities/{id} v0alpha2 adminDeleteIdentity
+// swagger:route DELETE /admin/identities/{id} identity deleteIdentity
 //
 // # Delete an Identity
 //
-// Calling this endpoint irrecoverably and permanently deletes the identity given its ID. This action can not be undone.
+// Calling this endpoint irrecoverably and permanently deletes the [identity](https://www.ory.sh/docs/kratos/concepts/identity-user-model) given its ID. This action can not be undone.
 // This endpoint returns 204 when the identity was deleted or when the identity was not found, in which case it is
 // assumed that is has been deleted already.
```

Comment:
Is this still true? The `Responses` section lists 404 as well.

Reply:
I'm not sure to be honest

Reply:
you're right!